stock_location_zone/models/picking_zone.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2017 Syvain Van Hoof (Okia sprl) <sylvainvh@okia.be>
+# Copyright 2017-2019 Jacques-Etienne Baudoux (BCIM sprl) <je@bcim.be>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class PickingZone(models.Model):
+    _name = 'picking.zone'
```

Comment:
The model is bothering me a bit, I would expect `stock.picking.zone` to be coherent with `stock.location`, `stock.picking.type`, ...

Reply:
me too. I agree. Should I then add migration file as it was named like that on v10 projects